java/src/jmri/jmrit/display/controlPanelEditor/EditCircuitFrame.java

```diff
@@ -190,14 +225,61 @@ private void convertIcons() {
      * *********************** end setup *************************
      */
 
+    private void createBlock() {
+        String userName = _blockName.getText().trim();
+        String systemName = _systemName.getText().trim();
+        OBlockManager mgr = InstanceManager.getDefault(OBlockManager.class);
+        StringBuilder  sb = new StringBuilder ();
+        if (userName.length() > 0) {
+             OBlock block = mgr.getByUserName(userName);
+            if (block != null) {
+                sb.append(Bundle.getMessage("duplicateName", userName, block.getSystemName()));
+                sb.append("\n");
+            }
+        }
+        if (!systemName.startsWith("OB") || systemName.length() < 3) {
```

Comment:
The OBlockManager has an inherited method `boolean isValidSystemNameFormat(String)` to check if a system name is a valid name. Please use it instead of implementing an alternate check.